docs/PublishingGuide.md

```diff
@@ -23,6 +22,20 @@
 It takes a few minutes for report generation. Meanwhile, you can monitor the progress live at [Travis-CI's Builds](https://travis-ci.org/dashboard/builds). <br/>
 Try accessing your site again when a green tick appears beside your fork.
 
+### Granting write-access to RepoSense for publishing
+We recommmend use of [personal access token](https://github.blog/2013-05-16-personal-api-tokens/) for ease of setup and [deploy key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) for enhanced security.
+
+##### If you wish to use `personal access token`:
+1. Follow this [guide](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) and give only `public_repo` permission <br/>
+1. **Copy** the token for later use <br/>
+
+##### If you wish to use `deploy key`:
```

Comment:
Do these instructions work for Windows users?

Reply:
Not with vanilla cmd. They would have to use online tools.

Reply:
Not sure, if it's a good idea to link users to online tools. Since these are mean to be created securely 😅.

Reply:
I see. can recommend alternatives for Windows users?